microsoft-edge/extensions-chromium/publish/sidebar.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Sidebar Feature
+description: A new sidebar feature is being added to the Microsoft Edge Browser where extensions can be displayed with their own UI.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 06/15/2023
+---
+# Sidebar Feature
+
+The sidebar feature of Microsoft Edge Add-Ons (extensions) enables users of your extension to view more information alongside the main content of a webpage.
+
+The _sidebar_ is a persistent column on the side of the browser, which exists alongside the primary content.  By enabling side-by-side browsing, the need to constantly switch tabs is reduced, and the user will have a more productive browsing experience.
+
+In the API, the sidebar is called a _side panel_.  In the UI, it's called a _sidebar_.
+
+As a developer, the sidebar API enables you to display your own UI on the web.  You can use the extension's content in the sidebar to supplement the user's primary task.
+
+![The sidebar for a Microsoft Edge extension](./sidebar-images/sidebar-screenshot.png)
+
+
+<!-- ====================================================================== -->
+## Features of Sidebar API
+
+* The sidebar has access to all the Microsoft Edge Extension APIs.
+* The sidebar remains open while navigating between tabs.
+* The sidebar is available on specific sites.
+
+Use this article to enable the sidebar for your new or existing extension.
+
+For complete details of how to create an extension, see:
+* [Create an extension tutorial, part 1](../getting-started/part1-simple-extension.md)
+* [Create an extension tutorial, part 2](../getting-started/part2-content-scripts.md)
+
+
+<!-- ====================================================================== -->
+## Origin
+
+As with other extension resources, the sidebar page commits to a trusted extension context on its origin (`extension://<id>`).  The sidebar has the same API access as other trusted extension contexts.
+
+All the existing extensions APIs are available for sidebar extensions, so you can leverage all current capabilities of the extensibility framework in your sidebar-enabled extension.
+
+To add a sidebar to your extension, follow the steps below.
+
+
+<!-- ====================================================================== -->
+## Step 1: Modify the manifest file
+
+Every extension for Microsoft Edge has a JSON-formatted manifest file, named `manifest.json`.  The manifest file is the blueprint of your extension.
+
+
+<!-- ------------------------------ -->
+#### Permissions and supported API
+
+The permission sidePanel should be included in the extension manifest file.
+
+Developers can use the supported API field, side_panel, to set the sidepanel option in their extension. 
```

Comment:
```suggestion
Use the `side_panel` API field to set the sidepanel option in your extension.
```

Reply:
locally changed the two sentences to:
Use the `side_panel` API field to set the side panel option in your extension.  Include the `sidePanel` permission in the extension manifest file: